app/test_main.py

```diff
@@ -1,3 +1,22 @@
 from app.main import get_human_age
 
-# write your code here
+import pytest
+
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (24, 24, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17]),
```

Comment:
Consider adding test cases for edge scenarios such as negative ages or very large numbers to ensure the function handles these cases correctly, as suggested in the checklist.